### DISCUSSION TITLE
Tobii 5 Support

### DISCUSSION THREAD

**Discussion by kamilgardelo** [opentrack/opentrack#1516] [Ideas]
Hi I saw some files in repo related with Tobii tracker. What about tobii 5 plugin?

**Comment by amadeus** [comment]
Curious about this as well. It appears some foundational code was added to support it, however there haven't been updates to that in a while

**Comment by GO63-samara** [comment]
If I'm not mistaken, @ballista-milsim was involved in connecting the tobii tracker:
https://github.com/opentrack/opentrack/commits?author=ballista-milsim

Maybe @ballista-milsim will answer?

**Comment by scottlincn** [comment]
Does this mean I can use opentrack for tobii?

**Comment by amadeus** [comment]
Not on it's own (as far as I know), you can however with FaceTrackNoIR as a bridge as outlined here: https://gist.github.com/amadeus/09b32e4d45f815f6d46686d367c4b6ee

**Reply by scottlincn** [reply]
I tried that before, but it is an inconvenient way to use it. So I hope opentrack can receive from Tobii directly.

**Reply by amadeus** [reply]
I agree, would love to be able to use it natively. Truth is, looks like the Tobii code that was merged, hasn't been touched in 1-2 years, so it's not looking good

**Reply by scottlincn** [reply]
I hope some kind person could invent a new Tobii code

**Comment by amadeus** [comment]
Tobii  support has been added to the most recent version of OpenTrack by the way!

**Reply by scottlincn** [reply]
Thanks for notification
